content/zh/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -1,5 +1,55 @@
 ---
-title: 调度和驱逐 (Scheduling and Eviction)
+title: 调度，抢占和驱逐
 weight: 90
-description: 在Kubernetes中，调度 (Scheduling) 指的是确保 Pods 匹配到合适的节点，以便 kubelet 能够运行它们。驱逐 (Eviction) 是在资源匮乏的节点上，主动让一个或多个 Pods 失效的过程。
+content_type: concept
+description: >
+  在Kubernetes中，调度 (scheduling) 指的是确保 Pods 匹配到合适的节点，
+  以便 kubelet 能够运行它们。抢占 (Preemption) 指的是终止低优先级的 Pods 以便高优先级的 Pods 可以
+  调度运行的过程。驱逐 (Eviction) 是在资源匮乏的节点上，主动让一个或多个 Pods 失效的过程。
 ---
+
+<!--
+---
+title: "Scheduling, Preemption and Eviction"
+weight: 90
+content_type: concept
+description: >
+  In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes
+  so that the kubelet can run them. Preemption is the process of terminating
+  Pods with lower Priority so that Pods with higher Priority can schedule on
+  Nodes. Eviction is the process of proactively terminating one or more Pods on
+  resource-starved Nodes.
+no_list: true
+---
+-->
+
+<!--
+In Kubernetes, scheduling refers to making sure that {{<glossary_tooltip text="Pods" term_id="pod">}}
+are matched to {{<glossary_tooltip text="Nodes" term_id="node">}} so that the
+{{<glossary_tooltip text="kubelet" term_id="kubelet">}} can run them. Preemption
+is the process of terminating Pods with lower {{<glossary_tooltip text="Priority" term_id="pod-priority">}}
+so that Pods with higher Priority can schedule on Nodes. Eviction is the process
+of terminating one or more Pods on Nodes.
+-->
```

Comment:
Did not quite translate this section as I feel like it overlaps with the description part above?

Reply:
please translate the above paragraph.